grouper/fe/templates/service.html

```diff
@@ -0,0 +1,318 @@
+{% block subheading %}
+    {{ user.username }}
+    {% if user.role_user %}<small>(role)</small>{% endif %}
+    {% if not user.enabled %}<small>(disabled)</small>{% endif %}
+
+    {% if group.audit and not group.audit.complete %}
+        <span class="label label-danger">AUDIT IN PROGRESS</i>
+    {% endif %}
+{% endblock %}
+
+{% block headingbuttons %}
+    <!-- Enable/Disable button. -->
+    {% if current_user_role['is_owner'] %}
+        {% if group.enabled %}
+            <button class="btn btn-danger" data-toggle="modal" data-target="#disableModal">
+                <i class="fa fa-minus"></i> Disable
+            </button>
+        {% else %}
+            <button class="btn btn-warning" data-toggle="modal" data-target="#enableModal">
+                <i class="fa fa-plus"></i> Enable
+            </button>
+        {% endif %}
+    {% endif %}
+
+    <!-- Approve members. -->
+    <div class="btn-group">
+        <a href="/groups/{{group.name}}/requests"
+           class="btn btn-default"><i class="fa fa-list"></i> Requests</a>
+        {% if current_user_role['is_approver'] and num_pending %}
+            <a href="/groups/{{group.name}}/requests?status=pending"
+               title="Pending Requests"
+               class="btn btn-default btn-pending">{{ num_pending }}</a>
+        {% endif %}
+    </div>
+
+    <!-- Show the join button if the user isn't already a member and the group
+         isn't invite-only. -->
+    {% if not current_user_role['is_approver'] and group.canjoin != "nobody" %}
+    <a href="/groups/{{group.name}}/join"
+       class="btn btn-success"><i class="fa fa-user"></i> Join</a>
+    {% endif %}
+
+    <!-- Add member or edit group -->
+    {% if current_user_role['is_approver'] %}
+        <a href="/groups/{{group.name}}/add"
+           class="btn btn-success"><i class="fa fa-plus-circle"></i> Add Member</a>
+        <a href="/groups/{{group.name}}/edit"
+           class="btn btn-primary"><i class="fa fa-edit"></i> Edit</a>
+    {% endif %}
+
+    <!-- Owners can't leave a group. -->
+    {% if not current_user_role['is_owner'] %}
+        <a href="/groups/{{group.name}}/leave"
+           class="btn btn-danger"><i class="fa fa-sign-out"></i> Leave</a>
+    {% endif %}
+{% endblock %}
+
+{% block content %}
+<div class="row">
+    <div class="col-md-6">
+        <blockquote><p>
+            <em>{{group.description|default("&nbsp;", True)|escape}}</em>
+        </p></blockquote>
+    </div>
+    <div class="col-md-6">
+        <h4>
+            {% if audited %}
+                <span class="label label-warning pull-right">
+                    <i class="fa fa-gavel"></i> <span {{ help_for("audited_group") }}>
+                        Auditing Enabled
+                    </span>
+                </span>
+            {% endif %}
+            {% if group.canjoin == "canjoin" %}
+                <span class="label label-success pull-right">
+                    <i class="fa fa-group"></i> Open Membership
```

Comment:
Maybe not at Dropbox, but I'm sure there is some use case in other organizations where one might want to have an SA group open to anyone